docs/getting-started-v2-beta.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page-with-side-nav
+title: Getting Started
+---
+# Getting Started
+
+De 'Bevraging Persoon' Web API is gespecificeerd in OpenAPI specifications (OAS).
+
+Wil je de API gebruiken? Dit kun je doen:
+
+1. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
+2. [Implementeer de API Client](#implementeer-de-api-client)
+3. [Probeer en test de API](#probeer-en-test-de-api)
+
+## Functionaliteit en specificaties
+
+Je kunt een visuele representatie van de specificatie genereren met [Swagger UI]({{ site.baseurl }}/swagger-ui) of [Redoc]({{ site.baseurl }}/redoc).
+
+De (resolved) OAS3 kun je hier downloaden: [openapi.yaml](https://github.com/VNG-Realisatie{{ site.baseurl }}/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml).
+
+De [functionele documentatie](./features) vind je in [features](./features).
+
+## Implementeer de API client
+Client code kun je genereren met de "[genereervariant](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml){:target="_blank" rel="noopener"}" van de API-specificaties en een code generator. Een overzicht met codegeneratoren kun je vinden op [OpenAPI.Tools](https://openapi.tools/#sdk){:target="_blank" rel="noopener"}.
+
+Deze repo bevat scripts waarmee je met [OpenAPI Generator](https://openapi-generator.tech/){:target="_blank" rel="noopener"} client code kunt genereren in JAVA, .NET (Full Framework & Core) en Python. De makkelijkste manier om de code generatie scripts te gebruiken, is door deze repo te clonen. Na het clonen kun je met `npm install` de benodigde packages installeren en kun je met npm run <script naam> één van de volgende scripts uitvoeren:
+- oas:generate-java-client (voor JAVA client code)
+- oas:generate-netcore-client (voor .NET Core client code)
+- oas:generate-net-client (voor .NET Full Framework client code)
+- oas:generate-python-client (voor Python client code)
+
+Een lijst met andere ondersteunde generator opties kun je vinden in de [Generators List](https://openapi-generator.tech/docs/generators){:target="_blank" rel="noopener"} van OpenAPI Generator.
+
+Note. De prerequisite van OpenAPI Generator is JAVA. Je moet een JAVA runtime installeren voordat je OpenAPI Generator kunt gebruiken
+  
+## Probeer en test de API
+
+Wil je de 'Bevraging Persoon' Web API proberen en testen? Kijk op: `https://www.haalcentraal.nl/haalcentraal/api/brp`
+
+Om de web api te gebruiken heb je een apikey nodig. Deze voeg je aan een request toe als header "X-API-KEY". Een API-key vraag je aan bij de product owner [c.dingemanse@comites.nl](mailto:c.dingemanse@comites.nl).
```

Comment:
, of wordt ter plekke aan jou uitgereikt op de API Labs.

Reply:
waneer je al eerder een apikey hebt gehad voor een Haal Centraal BRP API lab kan je die gebruiken.

@KayodeBakker dat gaat zo zijn toch? Dat men de oude api-key nog kan gebruiken op v2.0?

Reply:
aangepast